gemrb/core/EffectQueue.cpp

```diff
@@ -1031,11 +1031,11 @@ static int check_resistance(Actor* actor, Effect* fx)
 
 	//opcode immunity
 	// TODO: research, maybe the whole check_resistance should be skipped on caster != actor (selfapplication)
-	if (caster != actor && actor->fxqueue.HasEffectWithParam(fx_opcode_immunity_ref, fx->Opcode)) {
+	if (actor->fxqueue.HasEffectWithParam(fx_opcode_immunity_ref, fx->Opcode)) {
```

Comment:
The check was added to fix the statues from Watcher's keep level 1 from moving immediately, before the plot related scripting animates them. Does that still work fine?

Reply:
I plan to visit the Keep shortly in my current game, I'll check then.

Reply:
Okay, the statues are still and circleless, and (two of them) only attack after picking up the holy book.  Everything seems OK?  Two statues close to the entrance are doing their idle arms-waving animation, though.

Reply:
Do you mean the ogre and archer? The spawns are level dependent, but I couldn't get anyone to wave on master.

Reply:
Ogre and golem, actually.  And by waving I mean they're just wobbling them a little.

Reply:
Oh, yes, I see, they're indeed resisting HOLDRING's paralysis (opcode 109).  (The other statues don't.)  But they're not moving anyway, so maybe it's OK?

Reply:
Got it, this is just some mod adding paralyze immunity to all golems (and ogres I guess?)  Should be okay in vanilla.